### PR TITLE
Made related persons links loader, refactored pep models and serializers

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/views/pep_views.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from django_filters.rest_framework import DjangoFilterBackend
 from rest_framework import viewsets
 from rest_framework.filters import SearchFilter
+# from rest_framework.permissions import AllowAny
```
Comment (roman-tiukh) [inline business_register/views/pep_views.py:4]: ?

File: business_register/views/pep_views.py [hunk 2]
```diff
@@ -9,6 +10,7 @@
 
 
 class PepViewSet(CachedViewMixin, viewsets.ReadOnlyModelViewSet):
+    # permission_classes = [AllowAny]
```
Comment (roman-tiukh) [inline business_register/views/pep_views.py:13]: ?

File: business_register/serializers/company_and_pep_serializers.py [hunk 3]
```diff
@@ -188,7 +196,7 @@ class Meta:
 
 
 class PepDetailSerializer(DynamicFieldsMixin, serializers.ModelSerializer):
-    related_persons = RelatedPersonSerializer(many=True)
+    related_persons = PepShortestSerializer(many=True)
```
Comment (roman-tiukh) [inline business_register/serializers/company_and_pep_serializers.py:199]: i think we need to implement this via RelatedPersonSerializer

File: business_register/serializers/company_and_pep_serializers.py [hunk 4]
```diff
@@ -156,12 +166,10 @@ class Meta:
 
 
 class RelatedPersonSerializer(serializers.ModelSerializer):
+
```
Comment (roman-tiukh) [inline business_register/serializers/company_and_pep_serializers.py:169]: maybe need PepShortestSerializer here

File: business_register/serializers/company_and_pep_serializers.py [hunk 5]
```diff
@@ -96,14 +97,23 @@ class Meta:
...
-    # check_companies = CountFoundedCompaniesSerializer(many=True)
+    check_companies = CountFoundedCompaniesSerializer(many=True)
 
     class Meta:
         model = Pep
         fields = (
             'id', 'fullname', 'last_job_title', 'last_employer',
             'is_pep', 'pep_type', 'url',
-            # 'related_persons', 'related_companies', 'check_companies',
+            'related_persons', 'related_companies', 'check_companies',
```
Comment (roman-tiukh) [inline business_register/serializers/company_and_pep_serializers.py:107]: why you uncomment this?

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

**Comment by roman-tiukh** [comment]
first we need to decide what to do with the frontend

File: business_register/views/pep_views.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/views/pep_views.py:4]: for testing

Comment (IegorSoboliev) [reply business_register/views/pep_views.py:4]: for testing

File: business_register/views/pep_views.py (hunk 2 above)
Comment (IegorSoboliev) [reply business_register/views/pep_views.py:13]: for testing

File: business_register/serializers/company_and_pep_serializers.py (hunk 3 above)
Comment (IegorSoboliev) [reply business_register/serializers/company_and_pep_serializers.py:199]: done

File: business_register/serializers/company_and_pep_serializers.py (hunk 4 above)
Comment (IegorSoboliev) [reply business_register/serializers/company_and_pep_serializers.py:169]: done

File: business_register/serializers/company_and_pep_serializers.py (hunk 5 above)
Comment (IegorSoboliev) [reply business_register/serializers/company_and_pep_serializers.py:107]: deleted at all

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**